### PR TITLE
add vite config

### PR DESCRIPTION
**Pull request by awnigharbia**

### CODE REVIEW

**Review by john-d-pelingo — CHANGES_REQUESTED** [review]
I believe you can delete the `public` folder

File: package.json [hunk 1]
```diff
@@ -17,16 +19,18 @@
     "react-placeholder": "^3.0.2",
     "react-pose": "^4.0.8",
     "react-router-dom": "^5.0.1",
-    "react-scripts": "3.1.1",
     "react-spinners": "^0.6.1",
     "styled-components": "^4.3.2",
-    "typescript": "3.6.2"
+    "typescript": "3.6.2",
+    "vite": "^2.5.1",
+    "vite-plugin-svgr": "^0.3.0"
```
Comment (john-d-pelingo) [inline package.json:26]: Is this used anywhere?

Comment (awnigharbia) [reply package.json:26]: Currently no

Comment (john-d-pelingo) [reply package.json:26]: Then I suggest removing it

Comment (awnigharbia) [reply package.json]: Removed and pushed.

**Review by john-d-pelingo — APPROVED** [review]

**Pull request merged**